### ISSUE TITLE
Feature Request:  rvm-use Should Only Prompt for Gemset When Give the Prefix Arg

### ISSUE THREAD

**Issue by sshaw** [senny/rvm.el#55]
It seems that most of the time folks don't use a gemset, making this an unnecessary step (am I wrong?).

**Comment by senny** [comment]
I tend to agree. I only used rvm when gemsets were actively being used so I'm not the right person to know the current workflow. With `rbenv.el` there is no such step and I believe since bundler became default this is probably the same with rvm.
